README.md

```diff
@@ -534,6 +534,7 @@ This parameter corresponds to the `FlinkPulsarSource` in StreamAPI, the Properti
 | polltimeoutms | 120000 | Set the timeout for waiting to get the next message, in unit of milliseconds. | source |
 | failondataloss | true | When data is lost, the operation fails. | source |
 | commitmaxretries | 3 | Set the maximum number of retries when an offset is set for Pulsar messages. | source |
+| sendDelayMillisecond | 0 | delay millisecond message, just use **TableApi**, **StreamApi** use`PulsarSerializationSchema.setDeliverAtExtractor` | Sink |
```

Comment:
use `send-delay-millisecond`.  The configuration name in the with of flink sql will be converted to lowercase at runtime.